support/util/util.go

```diff
@@ -260,3 +260,23 @@ func IsIPv4(cidr string) (bool, error) {
 		return false, nil
 	}
 }
+
+// GetFirstUsableIP returns the first usable IP in both, IPv4 and IPv6 stacks.
+func GetFirstUsableIP(cidr string) (string, error) {
+	_, ipNet, err := net.ParseCIDR(cidr)
+	if err != nil {
+		return "", fmt.Errorf("error validating the incoming CIDR %s: %w", cidr, err)
+	}
+
+	// If we receive a CIDR which is not a network address
+	ip := ipNet.IP
+	if ip.To4() != nil {
+		// IPv4
+		ip[net.IPv4len-1]++
```

Comment:
both the ipv4 and ipv6 cases seem to be doing the same thing, so ip[len(ip)-1]++ should work for both, I don't think there's a need to differentiate.

Reply:
If I do that this is the result in the unitTests:

```
FirstUsableIP() = 2000::, want 2000::1
```

Reply:
I tested ip[len(ip)-1]++ for all cases and it passes for me